phoneApp/index.js

```diff
@@ -0,0 +1,195 @@
+class UsersContacts {
+	constructor() {
+		this.dataUsers = [
+		{
+			name: 'Иван',
+			lastName: 'Петров',
+			email: 'IvanPetrov@ec.ua' 
+		},
+		{
+			name: 'Сергей',
+			lastName: 'Сергей',
+			email: 'SergeiSergeev@ec.ua' 
+		},
+		{
+			name: 'Иван',
+			lastName: 'Иванов',
+			email: 'IvanIvanov@ec.ua' 
+		},
+		{
+			name: 'Александр',
+			lastName: 'Александров',
+			email: 'AlexAlex@ec.ua'
+		},
+		{
+			name: 'Алекс',
+			lastName: 'Смирнов',
+			email: 'AlexSmirnov@ec.ua'
+		},
+		{
+			name: 'Сергей',
+			lastName: 'Волков',
+			email: 'VolkovSergey@ec.ua' 
+		},
+		{
+			name: 'Мария',
+			lastName: 'Шарапова',
+			email: 'MariyaSharapova@ec.ua' 
+		},
+		{
+			name: 'Александр',
+			lastName: 'Винник',
+			email: 'AlexVinnik@ec.u'  
+		},
+		{
+			name: 'Дарий',
+			lastName: 'Смирнов',
+			email: 'DariySmirnov@ec.ua' 
+		},
+		{
+			name: 'Елена',
+			lastName: 'Лещенко',
+			email: 'ElenaLeshenko@ec.ua' 
+		},
+		{
+			name: 'Ольга',
+			lastName: 'Новикова',
+			email: 'OlgaNovikova@ec.ua' 
+		},
+		{
+			name: 'Наталья',
+			lastName: 'Шемякина',
+			email: 'ShemyakinaN@ec.ua' 
+		},
+		{
+			name: 'Анна',
+			lastName: 'Донцова',
+			email: 'AnnaDontsova@ec.ua' 
+		},
+		{
+			name: 'Влад',
+			lastName: 'Яма',
+			email: 'VladYama@ec.ua' 
+		},
+		{
+			name: 'Кира',
+			lastName: 'Воробьева',
+			email: 'Kira1990@ec.ua' 
+		},
+		{
+			name: 'Виктор',
+			lastName: 'Кривенко',
+			email: 'ViktorKriv@ec.ua' 
+		}	
+		];
+		this.columnHeadings = ['Name', 'Last name', 'Email'];
+	};
+
+	render() {
+
+		document.body.innerHTML += this.createHeader();
```

Comment:
document.body.innerHTML not sure I like to use document.body.

You have to create an additional tag at index.html and call it, for example,
```html
<div id="mountNode"></div>
```
And update content only inside such node. updating the whole document.body is too risky